### PR TITLE
Update the documentation

### PR DESCRIPTION
**Pull request by doomspork**
RFC @scrogson / @hassox — Thoughts on these changes?  Whatever we end up with here I'll apply to the docs for the other strategies so our documentation is consistent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,35 +2,82 @@
...
+    ```
 
-## Installation
+1. Add Google to your Überauth configuration:
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+    ```elixir
+    config :ueberauth, Ueberauth,
+      providers: [
+        google: [ { Ueberauth.Strategy.Google, [] } ]
```
Comment (scrogson) [inline README.md]: let's tighten up the spacing around the braces.

File: README.md [hunk 2]
```diff
@@ -2,35 +2,82 @@
...
+Or with options:
+
+    /auth/google?scope=email,profile
+
+By default the requested scope is "profile". Scope can be configured either explicitly as a `scope` query string value on the request path or in your configuration:.
+
+```elixir
+config :ueberauth, Ueberauth,
+  providers: [
+    google: { Ueberauth.Strategy.Google, [ default_scope: "emails,profile,plus.me" ]
```
Comment (scrogson) [inline README.md]: same here. (remove spacing around braces)

**Comment by doomspork** [comment]
Updated the spacing @scrogson.  Let me know if that's better.

**Comment by scrogson** [comment]
:+1:

**Comment by doomspork** [comment]
@scrogson / @hassox — Unless there's addition feedback this should be good to go.

**Pull request merged**